openmetadata-ui/src/test/java/org/openmetadata/catalog/selenium/events/Events.java

```diff
@@ -38,4 +38,27 @@ public static void sendEnter(WebDriver driver, By by) {
     (new WebDriverWait(driver, 15)).until(ExpectedConditions.elementToBeClickable(by));
     driver.findElement(by).sendKeys(Keys.ENTER);
   }
+
+  public static void sleep(int timeOutInSeconds) {
+    try {
+      Thread.sleep(timeOutInSeconds);
+    } catch (InterruptedException e) {
+      e.printStackTrace();
+    }
+  }
+
+  public static boolean waitForElementToDisplay(WebDriver driver, By by, int timeOutInSeconds) {
```

Comment:
@kshinde2512 why can't we use the option described here https://stackoverflow.com/questions/20903231/how-to-wait-until-an-element-is-present-in-selenium
also, this looks interesting. https://selenide.org/index.html cc @vivekratnavel